msal/consumer-rules.pro

```diff
@@ -17,7 +17,7 @@
 #}
 
 ##---------------Begin: proguard configuration for MSAL  --------
--keep class com.microsoft.** { *; }
+-keep class com.microsoft.identity.client.** { *; }
```

Comment:
I think we could keep these instead.
 com.microsoft.identity.**
 com.microsoft.device.**
 
would that be good enough for your app?

Reply:
```suggestion
-keep class com.microsoft.identity.** { *; }
```
I have added the device rule in another PR for the common lib
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1047

Reply:
would this be enough or do you want the rule to be duplicated here?